cinema/tests/test_movie_api.py

```diff
@@ -157,3 +158,179 @@
         res = self.client.get(MOVIE_SESSION_URL)
 
         self.assertIn("movie_image", res.data[0].keys())
+
+
+class UnauthenticatedCinemaAPITests(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+
+    def test_auth_required(self):
+        res = self.client.get(MOVIE_URL)
+        self.assertEqual(res.status_code, status.HTTP_401_UNAUTHORIZED)
```

Comment:
This test case is not clear. It's better to give a meaningful name to the test case that describes what it is testing. For example, it could be `test_unauthenticated_user_cannot_access_movie_list` or something similar.